services/hanke-service/src/main/resources/db/changelog/changesets/076-create-hankkeen-haittojenhallintasuunnitelma-table.sql

```diff
@@ -0,0 +1,20 @@
+--liquibase formatted sql
+--changeset Teemu Hiltunen:076-create-hankkeen-haittojenhallintasuunnitelma-table
+--comment: Create a table for project's nuisance management plan
+
+CREATE TABLE hankkeen_haittojenhallintasuunnitelma
+(
+    hankealue_id        bigint                   NOT NULL,
+    tyyppi              varchar(255)             NOT NULL,
```

Comment:
These should probably make a unique key. There shouldn't be two different plans for how to handle cycle traffic in one area.

Reply:
Should there be a primary key in this table?

Reply:
Yes, you're right. I added a combined PK. And also renamed "nuisance management" to "nuisance control" in comments and docs as that is how it is translated in UI.